anthologize/README.md

```diff
@@ -22,48 +20,49 @@ You'll need Python >=3.5 and the Python packages `latexcodec` and `pybtex`; thes
 
 ## 2. Assembling the data
 
-ACLPUB is written primarily with Softconf's STARTv2 system in mind.
+If you are using:
+
+- STARTv2: follow the instructions in Step 2(a), then 2(c).
+- EasyChair: follow the instructions in Step 2(b), then 2(a) and 2(c).
+- Another system: you will need to manually arrange your files into the layout described in Step 2(c).
```

Comment:
Or should users of other systems directly create the file format in Step 3?

Reply:
I am actually thinking of removing step 3 and just doing that myself.

Reply:
I want to give this some more thought, though, so I'm going to delay it a week till after the ACL deadline.

Reply:
We can try to get the documentation in shape for the software as it stands, though, and if/when the software changes, the documentation should change with it.